src/main/java/com/radixdlt/serialization/ClassScanningSerializerIds.java

```diff
@@ -136,6 +139,17 @@ private void collectSupertypes(Class<?> cls) {
 		}
 	}
 
+	private void collectInterfaces(Class<?> cls) {
+		var interfaces = Stream.of(cls.getInterfaces())
+				.filter(this::isSerializerRoot)

```

Comment:
Would prefer to see a single tab here, rather than two.

Reply:
Done